### DISCUSSION TITLE
more details about 'Connection seems to be dead'?

### DISCUSSION THREAD

**Discussion by bethebest0622** [curl/curl#12733] [General]
i think 'seems' is not clear enough.

could you add more details about why the connections are dead?

i use the self-complied version, which only can dive into the https://github.com/curl/curl/blob/master/lib/url.c#L816

the dead happened when the connections are idle for about 10 minutes, even i set the MAX_CONN_AGE very large(7200L).

could you provide some tips about how to maintain the connections as long as i can?

for more information: the counterpart is Binance, is this may caused by Binance Server side close? if so, how can i confirm that?

**Comment by jay** [comment]
Didn't we just discuss this in #12715? If the server closes the connection there's nothing curl can do about it.

**Reply by bethebest0622** [reply]
hi, jay. good to see u again.

i think this is another discussion.

when the connections dead, i am wondering how to confirmed the dead reason. it may be caused by server close it, or some other reason, server close connection is more than we can control.

but, if there is more information can be found in verbose(not just Connection #1 seems to be dead), we can figure out how to improve our program.

**Reply by jay** [reply]
> when the connections dead, i am wondering how to confirmed the dead reason. it may be caused by server close it, or some other reason, server close connection is more than we can control.

Do you not see that information when enabling the tracing data and verbose mode as [mentioned](https://github.com/curl/curl/discussions/12715#discussioncomment-8152625) in the other thread?
